source/index.js

```diff
@@ -25,8 +25,9 @@ const exec = (cmd, args) => {
 	const cp = execa(cmd, args);
 
 	return merge(
-		streamToObservable(cp.stdout.pipe(split()), {await: cp}),
-		streamToObservable(cp.stderr.pipe(split()), {await: cp})
+		streamToObservable(cp.stdout.pipe(split())),
+		streamToObservable(cp.stderr.pipe(split())),
+		cp
```

Comment:
`merge()` is not documented to accept promises, unless I'm mistaken?

https://www.learnrxjs.io/operators/combination/merge.html